repo/compositeArtifacts.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?compositeArtifactRepository version='1.0.0'?>
+<repository name='GWT Eclipse Plugin'
+    type='org.eclipse.equinox.internal.p2.artifact.repository.CompositeArtifactRepository' 
+    version='1.0.0'>
+  <properties size='1'>
+    <property name='p2.timestamp' value='1665755219001'/>
+  </properties>
+  <children size='2'>
+    <child location='https://gwt-plugins.github.io/documentation/repo/gwt-eclipse-plugin'/>

```

Comment:
I'm not clear if we're talking about documentation or plugins.
I'm afraid you intend to move the plugin storage to github pages which... looks like a dangerous choice.

Reply:
I changed the location to plugins.gwtproject.org/eclipse in the second commit as discussed with colin.
Nothing will be on the github.io page except the documentation itself.

Reply:
Go ahead, then!